app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -102,6 +107,10 @@ If you're not sure which route into teaching is right for you, [find routes into
 
 $check-your-qualifications$
 
+$what-teaching-taught-me$
```

Comment:
This works fine but is a little odd as it feels a bit shoehorned on the page

Does it need an h2 or h3 above it?